src/lib/responses/error.response.ts

```diff
@@ -0,0 +1,22 @@
+import { Response } from "express";
+
+interface ErrorInterface {
+    error: string,
+    statusCode: number,
+    timestamp: number
```

Comment:
let's not worry about adding a timestamp here for now. We're working on adding proper logging to all services, and the timestamp will be added automatically by the structure logger we're using.